core/src/test/java/com/graphhopper/routing/util/EncodingManagerTest.java

```diff
@@ -75,12 +75,13 @@ public void testSupportFords() {
 
     @Test
     public void validEV() {
-        for (String str : Arrays.asList("blup_test", "test", "test12", "tes$0", "car_test_test", "small_car$average_speed")) {
+        for (String str : Arrays.asList("blup_test", "test", "test12", "car_test_test")) {
             assertTrue(EncodingManager.isValidEncodedValue(str), str);
         }
 
-        for (String str : Arrays.asList("Test", "12test", "test|3", "car__test", "blup_te.st_", "car___test", "car$$access",
-                "test{34", "truck__average_speed", "blup.test", "test,21", "täst", "blup.two.three", "blup..test")) {
+        for (String str : Arrays.asList("Test", "12test", "test|3", "car__test", "small_car$average_speed", "tes$0",
+                "blup_te.st_", "car___test", "car$$access", "test{34", "truck__average_speed", "blup.test", "test,21",
+                "täst", "blup.two.three", "blup..test")) {
```

Comment:
Does this mean any $ sign is forbidden in custom models now? Was it forbidden before?